keyboards/levinson/keymaps/jyh2/config.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This file esn't necessary, and wont get processed.

Reply:
Sorry, that's incorrect. 

But this file has all of the same issues as the other, and should be updated in the same way